charts/enterprise/authelia/values.yaml

```
@@ -249,7 +249,7 @@ authentication_backend:
     ##
     ## To allow sign in both with username and email, one can use a filter like
     ## (&(|({username_attribute}={input})({mail_attribute}={input}))(objectClass=person))
-    users_filter: ""
+    users_filter: "(&(|({username_attribute}={input})({mail_attribute}={input}))(objectClass=person))"
```

```suggestion
    users_filter: ""
```
No default needed in values.yaml at all. See notes